Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+FROM registry.ci.openshift.org/openshift/release:golang-1.19 AS builder
```

Comment:
I wondering why the ART bot didn't update this?

Reply:
ART builds use Dockerfile.rhel. Not sure if this Dockerfile is used anywhere right now.